docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+* Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
Now that we've dealt with character number 1, we can move onto the rest :wink:  Can it be changed like this:

Fixed a bug where ~~the~~ :func:`iris.util._is_circular` would *erroneously* return false ~~where~~ when ~~the~~ coordinate values are decreasing.